static/js/containers/DashboardPage.js

```diff
@@ -126,6 +135,26 @@ class DashboardPage extends React.Component {
     }
   };
 
+  updateRequirements = () => {
+    this.fetchDashboard();
+    this.fetchCoursePrices();
+    this.handleOrderStatus();
+  };
+
+  fetchDashboard() {
+    const { dashboard, dispatch } = this.props;
+    if (dashboard.fetchStatus === undefined) {
```

Comment:
TIL too, I was going to argue for it for stylistic reasons and then I found evidence to support my preference :)